conans/requirements.txt

```diff
@@ -3,7 +3,7 @@ requests>=2.8.1, <3.0.0
 urllib3!=1.25.4,!=1.25.5
 colorama>=0.3.3, <0.5.0
 PyYAML>=3.11, <6.0
-patch-ng==1.17.2
+patch-ng>=1.17, <1.18
```

Comment:
But I am afraid this might resolve to patch-ng 1.17.0 or 1.17.1 that actually contain some bugs or old behavior we want to avoid?

Reply:
It won't update the version if you don't request it as a user and the one you already have installed satisfies the limits. If you install Conan from zero, it will take the last version available.

Pros:
 * if we fix something and release the `patch-ng` version, we don't need to wait for a new Conan release.

Cons:
 * people won't have the last version installed, but it is what we are doing right now with other requirements... do people need the last version? For example, right now, the last bug/patch is only affecting to one user.
   ...but we always have the possibility of bumping minor component if we want everyone to update the version

IMO, if it is a different library, we should add some freedom to it.

Reply:
Oh! Or your concern is just because of the lower limit? I'm changing it now, it should be considered a bug in this PR, not intended

Reply:
patch-ng is bug proof, there won't be a new release 😂 Maybe you could use the latest version as minimal version:

`patch-ng>=1.17.4, <1.18`

Reply:
Yes, exactly, the lower limit, I think patch-ng 1.17.0/1 will not work with current Conan.